scripts/updatebom.sh

```diff
@@ -17,22 +17,26 @@ set -euo pipefail
 
 source ./scripts/test_lib.sh
 
-function bom_fixlet {
+function bom_fix {
   log_callout "generating bill-of-materials.json"
 
   cp go.mod go.mod.tmp
   cp go.sum go.sum.tmp
 
-  local modules
-  # shellcheck disable=SC2207
-  modules=($(modules_for_bom))
+  local _bom_modules=()
+  load_workspace_relative_modules_for_bom _bom_modules
+  #
+  # Intentionally run the command once first, so it fetches dependencies. The exit code on the first
+  # run in a just cloned repository is always dirty.
+  GOOS=linux run_go_tool github.com/appscodelabs/license-bill-of-materials \
+    --override-file ./bill-of-materials.override.json "${_bom_modules[@]}" &>/dev/null
```

Comment:
Does `go work sync` work?

Reply:
Yes, I agree. The problem is that:

1. The repository: https://github.com/appscodelabs/license-bill-of-materials doesn't have issues/discussion enabled.
2. The last commit to that repo is from three years ago.
3. We want to replace this with https://github.com/etcd-io/etcd/issues/18902. So, I think the workaround now should be fine.